source/base/polynomial.cc

```diff
@@ -1395,34 +1393,13 @@ namespace Polynomials
         // entries given below. For degree 4 the eigenvalue is zero, so bypass
         // the LAPACK logic in that case.
```

Comment:
I think this comment needs to be updated since the function no longer depends on LAPACK.